tests/modules/picard/liftovervcf/test.yml

```diff
@@ -0,0 +1,13 @@
+- name: picard liftovervcf test_picard_liftovervcf
+  command: nextflow run tests/modules/picard/liftovervcf -entry test_picard_liftovervcf -c tests/config/nextflow.config
+  tags:
+    - picard/liftovervcf
+    - picard
+  files:
+    - path: output/picard/test.lifted.vcf.gz
+      contains:
+        - "chr22"
+    - path: output/picard/test.unlifted.vcf.gz
+      md5sum: 43d760d50cb49e461c9fc3949fa4fc03
+    - path: output/picard/versions.yml
+      md5sum: b16383e5111c1850cb2c8f97353001d0
```

Comment:
MD5sum Can be removed here to make it easier to update the module in the future with version bumps,

Reply:
@Lucpen

Reply:
@jfy133 should I simply delete the md5sum or replace them with something else like contains?

Reply:
You can just delete, not a problem. As long as the file is there it should have worked :+1: